### DISCUSSION TITLE
Jest Tests fails after upgrading to 9.3, cannot find interopRequireDefault

### DISCUSSION THREAD

**Discussion by olaj** [vercel/next.js#11219] [Help] [answered]
I have a inherited next.js project. I've tried to upgrade to 9.3.1 yesterday but when i tried to run my tests it just fails straightway with this error message.

Cannot find module '@babel/runtime-corejs2/helpers/interopRequireDefault' from 'jest.setup.ts'

I guess it has something todo with that performance optimizing thing that has been done? But i can't figure out what to do.

As i said the project is not mine from the beginning so I'm not 100% sure exactly how everything is setup but we are using Jest, Enzyme and React Test Render. 

The full message is:

`    Cannot find module '@babel/runtime-corejs2/helpers/interopRequireDefault' from 'jest.setup.ts'

      1 | import { configure } from "enzyme";
      2 | import Adapter from "enzyme-adapter-react-16";
    > 3 | 
        | ^
      4 | import {GlobalWithFetchMock} from "jest-fetch-mock";
      5 | 
      6 | const customGlobal: GlobalWithFetchMock = global as GlobalWithFetchMock;

      at Resolver.resolveModule (node_modules/jest-resolve/build/index.js:276:11)
      at Object.<anonymous> (jest.setup.ts:3:30)`

**Comment by timneutkens** [comment, marked as answer]
You will probably want to run this:

```
jest --clearCache
```

Jest caches transforms aggressively and we updated the next/babel preset a bit (to reduce bundle size).

**Reply by Janpot** [reply]
👍 Ok makes sense, I guess my messing with my `jest.config.js`/`.babelrc` invalidated its cache as well then.

**Reply by timneutkens** [reply]
Yep that would make sense too 👍

**Comment by olaj** [comment]
Damn, that simple. That is two hours i will never get back :D Hopefully this thread can help others 😄

**Reply by embryCODE** [reply]
Sonofa! Yeah, me too. Welp, guess if I would have googled better I could have found this sooner.
